internal/client/config.go

```diff
@@ -183,10 +197,16 @@ func (c *Config) GetContext(n string) (*api.Context, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	if c, ok := cfg.Contexts[n]; ok {
 		return c, nil
 	}
 
+	if n == incluster {
+		nc := c.newInclusterContext()
```

Comment:
no need to create a var here just return it

Reply:
Ok.